tests/light_client_tests.move

```diff
@@ -333,17 +323,14 @@ fun test_insert_header_failed_timestamp_too_old() {
     );
     let h = *lc.head();
     lc.insert_header(&h, new_header);
-    sui::test_utils::destroy(lc);
-    scenario.end();
+    abort
 }
 
-#[test]
-#[expected_failure(abort_code = EAlreadyUpdated)]
+#[test, expected_failure(abort_code = EAlreadyUpdated)]
 fun test_update_version_fail() {
     let sender = @0x01;
     let mut scenario = test_scenario::begin(sender);
     let mut lc = new_lc_for_test(scenario.ctx());
     update_version(&mut lc);
-    sui::test_utils::destroy(lc);
-    scenario.end();
+    abort
```

Comment:
I think its better to destroy the lc and end the scenario. We should treat it as any other test case i think. Unless there is some guide that we should abort in this scenario i prefer to keep the previous version

Reply:
In this case abort is clean. Because we expected error here and when test not fail abort return different error code we expected.